var/spack/repos/builtin/packages/rmlab/package.py

```diff
@@ -47,3 +47,12 @@ class Rmlab(CMakePackage):
     conflicts('%pgi@:14')
 
     depends_on('pngwriter@0.6.0:', when='+png')
+
+    def cmake_args(self):
+        spec = self.spec
+
+        args = [
+            '-DRmlab_USE_PNG={0}'.format((
+                'ON' if '+png' in spec else 'OFF'))
```

Comment:
I don't think you need to use double parentheses here, single parentheses should suffice.

Reply:
thx, fixed! :)